### PR TITLE
Fix prescreen 1B CSS 

### PR DESCRIPTION
**Pull request by fords**

### CODE REVIEW

File: src/pages/prescreen-1b.tsx [hunk 1]
```diff
@@ -58,207 +67,284 @@ class Prescreen1b extends React.Component<Prescreen1bProps> {
...
-          <GlosaryStyle>
-          <Glossary 
-          title="MYSOCIALSECURITY"
-          link="https://www.ssa.gov/myaccount/"
-          linkText="Login or signup online for a MySocialSecurity using this link."
-          >
-          MySocialSecurity is the Social Security Administrations online service. With a MySocialSecurity account , you can download a copy of your earnings record to use for this question.
-          </Glossary>
-          {
-            this.showFileUpload() == true && <Glossary
```
Comment (thadk) [inline src/pages/prescreen-1b.tsx:237]: The 2 glossary entries below got deleted accidentally I think

Comment (fords) [reply src/pages/prescreen-1b.tsx:237]: I added them back

Comment (fords) [reply src/pages/prescreen-1b.tsx]: If you want to update the current images, please feel free to update them. The resolution could be better.

Comment (fords) [reply src/pages/prescreen-1b.tsx:237]: If you want to update the current images in a carousel, please feel free to update them. The resolution could be better.

**Comment by thadk** [comment]
image
Thanks for following up on Slack, Zey. The results page is still having the issue with the glossary falling down I think. If you can rebase to the latest version on develop, that would be helpful too. @misspran made some modification and it may have changed the behavior a little.

**Review by thadk — CHANGES_REQUESTED** [review]
See comment above: Try rebasing to develop and see if you can address the glossary that falls down.

**Review by thadk — COMMENTED** [review]
Fixed it up for mobile and glossary.

**Pull request merged**